src/uct/ib/base/ib_md.c

```diff
@@ -663,6 +663,10 @@ static uint64_t uct_ib_md_access_flags(uct_ib_md_t *md, unsigned flags,
 {
     uint64_t access_flags = UCT_IB_MEM_ACCESS_FLAGS;
 
+    if (IBV_DEV_ATTR(&md->dev, max_qp_rd_atom) == 0) {
+        access_flags &= ~IBV_ACCESS_REMOTE_ATOMIC;
+    }
```

Comment:
1. this parameter is for both rdma_read and atomics, so can't use it to "detect" atomics support
2. need to check "atomic_cap" field
3. step (2) need to happen only on "verbs" memory domain since for mlx5 it may be not indicative

Reply:
Actually, this is good for me... would you agree if this field is 0 we should disable remote reads/writes as well? 
````
if (IBV_DEV_ATTR(&md->dev, max_qp_rd_atom) == 0) {
        access_flags = IBV_ACCESS_LOCAL_WRITE;
}
````

Reply:
maybe disable REMOTE_READ only
what does IB spec say about the value of this field?

Reply:
IB spec doesn't address `max_qp_rd_atom`, did you mean PRM?

Reply:
it's called "Max Responder Resources" in the spec
anyway, why not check atomic_cap directly to indicate atomics support?

Reply:
also, looking again at the spec - rdma read has to be supported by RC. so the common code should not disable rdma read.